app/Actions/Influxdb/v2/BuildPointData.php

```diff
@@ -48,7 +48,9 @@ public function handle(Result $result): Point
             ->addField('upload_latency_avg', Number::castToType(Arr::get($result->data, 'upload.latency.iqm'), 'float'))
             ->addField('upload_latency_high', Number::castToType(Arr::get($result->data, 'upload.latency.high'), 'float'))
             ->addField('upload_latency_low', Number::castToType(Arr::get($result->data, 'upload.latency.low'), 'float'))
-            ->addField('packet_loss', Number::castToType(Arr::get($result->data, 'packetLoss'), 'float'));
+            ->addField('packet_loss', Number::castToType(Arr::get($result->data, 'packetLoss'), 'float'))
+            ->addField('downloaded_bytes', Number::castToType(Arr::get($result->data, 'download.bytes'), 'int'))
```

Comment:
Since we're going to write the `bytes` to a column let's reference that instead of the `data` array.